jobs/promptLicenceCreation.ts

```diff
@@ -11,29 +11,28 @@ import applicationInfo from '../server/applicationInfo'
 initialiseAppInsights(applicationInfo('create-and-vary-a-licence-prompt-licence-create-job'))
 
 import logger from '../logger'
-import { Prisoner } from '../server/@types/prisonerSearchApiClientTypes'
 import config from '../server/config'
 import { services } from '../server/services'
 import { ManagedCase } from '../server/@types/managedCase'
 import LicenceStatus from '../server/enumeration/licenceStatus'
-import { EmailContact } from '../server/@types/licenceApiClientTypes'
+import type { CvlPrisoner, EmailContact } from '../server/@types/licenceApiClientTypes'
 import { convertToTitleCase } from '../server/utils/utils'
 import PromptLicenceCreationService from './promptLicenceCreationService'
 
-const { caseloadService, prisonerService, communityService, licenceService } = services
+const { caseloadService, communityService, licenceService } = services
 
-const promptLicenceCreationService = new PromptLicenceCreationService(prisonerService, caseloadService)
+const promptLicenceCreationService = new PromptLicenceCreationService(licenceService, caseloadService)
 
 const buildEmailGroups = async (
   urgentPromptCases: ManagedCase[],
   initialPromptCases: ManagedCase[]
 ): Promise<EmailContact[]> => {
   const managedCases = [...urgentPromptCases, ...initialPromptCases]
 
-  const mapPrisonerToReleaseCase = (prisoner: Prisoner) => {
+  const mapPrisonerToReleaseCase = (prisoner: CvlPrisoner) => {
     return {
       name: convertToTitleCase(`${prisoner.firstName} ${prisoner.lastName}`),
-      crn: prisoner.crn,
+      crn: undefined as string, // FIXME - this is never worked!
```

Comment:
ticket

Reply:
Does this make the crn `"undefined"` or ` `?

Reply:
Haven't followed it all the way through - just checked and it never was returned from prisoner offender search

Reply:
Looks like it won't populate this: `"${prisoner.name} (CRN: ${prisoner.crn}), who is due to leave custody on`

Reply:
So I assume it would say:

`Joe Bloggs (CRN: undefined), who is due to leave custody on ...'?